fs/xfstests.py

```diff
@@ -82,7 +82,7 @@ def setup_nvdimm(self):
                     mount = False
                 else:
                     self.cancel('Need %s GB to create loop devices' % check)
-                self._create_loop_device(None, '2038M', mount)
+                self._create_loop_device('2038M', mount)
```

Comment:
@disgoel hardcoding disk size incode, please see if you can pass it from yaml... as available disk space per system varies.. so user can test with smaller or bigger loop device

Reply:
@abdhaleegit 
looks like legacy region ndctl is not supported on powerpc, this is supported on x86.
In this condition we are using loop device as a log device and xfs restrict max log size to 2136997888, which is 10M less than 2GB. So we need to set loop_size as 2038M here.